### DISCUSSION TITLE
Change monetary policy from fithening to halving

### DISCUSSION THREAD

**Discussion by stevanlohja** [ethereumclassic/ECIPs#491] [ECIPs]
The current block reward is reduced by 20% every 5 ml blocks. What are you thoughts about updating the schedule to reduce the block reward by 50% rather than 20%?

[UPDATES]
1. [Unveiling ECIP-1017: The Ambitious Motivations Behind Ethereum Classic's Monetary Policy - PART 1](https://stevcrypto.substack.com/p/unveiling-ecip-1017-the-ambitious) 02/14/2024
2. (_Coming Soon_) Unraveling the Reality: Assessing the Impact of ECIP-1017 on Ethereum Classic's Cryptoeconomic Landscape - PART 2

**Comment by bobsummerwill** [comment]
I think that any such change is exceedingly unlikely.  Fixed monetary policy is a keystone for ETC.

That aside, from a personal POV, I  think it would probably be better to have a smooth curve function, rather than these sharp junction points.  But that is just an intuition.

Ultimately, though, I think that the most important element here is the predictability and having an unchanging monetary policy.  Which we have, for better or worse WRT the details.

**Reply by ghost** [reply]
To make such a drastic change there needs to be a good reasoning out of necessity for it, that's a part I missed in your proposal Stev. So I tend to agree with Bob here.

**Reply by stevanlohja** [reply]
> I think that any such change is exceedingly unlikely. Fixed monetary policy is a keystone for ETC.

I think it could be probable because it's an incentive for majority of ecosystem participants. Ethereum Classic did not originally have a controlled supply, but [ECIP-1017](https://ecips.ethereumclassic.org/ECIPs/ecip-1017) specifically describes a controlled supply model that changed the original monetary policy. A halving would make ECIP-1017 more assertive.

> To make such a drastic change there needs to be a good reasoning out of necessity for it

Thought challenge: I don't think is that drastic, neither the current status quo of 20%. What was the "necessity" for ECIP-1017?

**Reply by OmniEdge** [reply]
ETH did not have a clear monetary policy. From start they said it was wip. Kick the can down the road culture....

**Reply by stevanlohja** [reply]
@OmniEdge What was the "necessity" for ECIP-1017?

**Reply by OmniEdge** [reply]
Having finality in our Monetary policy with clarity and predictability. ECIP-1017 is clear on its motivation to secure and nurture the network.  Read it.

**Reply by stevanlohja** [reply]
I respect a lot of people may believe that but ETC supply inflation is 79% higher than the original forecast for ECIP-1017.

**Reply by OmniEdge** [reply]
Are you willing to share that data your believe is based on?

**Reply by stevanlohja** [reply]
> Are you willing to share that data your believe is based on?

Sure, but probably share in a different format and link here.

Anyone can verify the supply inflation through many public resources in the meantime 🕵️‍♂️

**Comment by OmniEdge** [comment]
Agree with Bob and W1g0. All participants in ETC have currently a less aggressive (compared to 50%) monetary policy. This gives miner less pressure to adjust to that change, more time for the ecosystem to develop/grow towards a healthier block fee market.

Any reason to increase to 50% ?

**Reply by stevanlohja** [reply]
> Any reason to increase to 50% ?

There were notable challenges and aspects that didn't yield the intended outcomes with ECIP-1017. So, this presents an opportunity to make ECIP-1017 more robust.

**Reply by OmniEdge** [reply]
What challenges and aspects? Putting more pressure on miners is not robust. We have a healthy hashrate so miners still satisfied with block rewards. It's working and giving us enough time to build on top of ETC giving ETC time to increase fee revenue per block. That's where it will go. Fees will increase with more adoption and then we will have the same realities as Bitcoin. We have a clear, consistent and predictable monetary policy. We will need to prepare for l2 scalable solutions with cheaper fees rather then trying to behave like the FED. We need more uses case built on top of ETC making more txs. That's is what is needed and makes ETC more robust.

**Comment by gitr0n1n** [comment]
Nice troll @stevanlohja . I hope you're enjoying Cardano.

Simple answer to this: Please go and review the ECIP 1017 discussions that established a `fixed monetary policy` for ETC.

https://ecips.ethereumclassic.org/ECIPs/ecip-1017

**Reply by stevanlohja** [reply]
It's a controlled/ fixed supply model, not a fixed monetary policy.
